types/react-fa/index.d.ts

```diff
@@ -37,7 +37,6 @@ export const Icon: ComponentClass<IconProps>;
 
 export interface IconStackProps extends SizeOverrideHTMLProps<IconStack> {
     size?: IconSize;
-    children?: IconProps[];
```

Comment:
for context, this is the error being generated with your proposed changes:
```
TS2324: Property 'children' is missing in type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<IconStackProps, ComponentState>> & Reado...'.
```
(unable to get the full error text for some reason)